Command Console: Failure to set subsystem properties if subsystem restart involves read-write transaction (Enterprise-only)

If setting a configuration property for a subsystem and its subsequent restart involves the use of a (nested) read-write transaction, the operation fails and can leave the system in an inoperable state. This is an Enterprise-only issue.

Steps to reproduce
- navigate to Command Console
- activate subsystems plugin (`activatePlugin subsystems`)
- change authentication chain (`setProperty Authentication;manager "chain=alfrescoNtlm2:alfrescoNtlm"`)

The operation fails with the following stacktrace:

```
Caused by: org.alfresco.error.AlfrescoRuntimeException: 02190037 Read-Write transaction started within read-only transaction
        at org.alfresco.repo.transaction.RetryingTransactionHelper.doInTransaction(RetryingTransactionHelper.java:371) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.repo.transaction.RetryingTransactionAdvice.invoke(RetryingTransactionAdvice.java:74) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.0.14.jar:6.0.14]
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:244) ~[spring-aop-6.0.14.jar:6.0.14]
        at jdk.proxy3.$Proxy168.removeAttribute(Unknown Source) ~[?:?]
        at org.alfresco.enterprise.repo.management.subsystems.PropertyBackedBeanAdapter.purge(PropertyBackedBeanAdapter.java:752) ~[alfresco-enterprise-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.enterprise.repo.management.subsystems.PropertyBackedBeanExporter$1.onApplicationEvent(PropertyBackedBeanExporter.java:216) ~[alfresco-enterprise-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.repo.management.subsystems.DefaultPropertyBackedBeanRegistry.broadcastEvent(DefaultPropertyBackedBeanRegistry.java:173) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.repo.management.subsystems.DefaultPropertyBackedBeanRegistry.deregister(DefaultPropertyBackedBeanRegistry.java:93) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.repo.management.subsystems.AbstractPropertyBackedBean.destroy(AbstractPropertyBackedBean.java:593) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
        at org.alfresco.repo.management.subsystems.ChildApplicationContextFactory.destroy(ChildApplicationContextFactory.java:404) ~[alfresco-repository-23.2.2.1.jar:23.2.2.1]
...
```

This issue is due to the fact that ACS Enterprise is trying to remove the DB persisted properties during the shutdown of the subsystem.

--- a/repository/src/main/resources/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/command-console-plugins/subsystem-commands.post.desc.xml
+++ b/repository/src/main/resources/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/command-console-plugins/subsystem-commands.post.desc.xml
@@ -17,5 +17,5 @@
     <negotiate accept="application/json">json</negotiate>
     <authentication>admin</authentication>
     <lifecycle>internal</lifecycle>
-    <transaction bufferSize="8192" allow="readonly">required</transaction>
+    <transaction bufferSize="8192" allow="readwrite">required</transaction>
 </webscript>

